editor/editor_node.cpp

```diff
@@ -7073,63 +7073,65 @@ EditorNode::EditorNode() {
 
 	// More visually meaningful to have this later.
 	raise_bottom_panel_item(AnimationPlayerEditor::get_singleton());
-	add_editor_plugin(memnew(ReplicationEditorPlugin));
 
 	add_editor_plugin(VersionControlEditorPlugin::get_singleton());
```

Comment:
I'd suggest changing that one too to follow the same pattern as others.
It implements the singleton pattern this way, which is not a bad idea, but it's inconsistent with the common usage in Godot:
```cpp
VersionControlEditorPlugin *VersionControlEditorPlugin::get_singleton() {
        return singleton ? singleton : memnew(VersionControlEditorPlugin);
}
```

Make that just `return singleton;` and you can `memnew` it here instead, like other plugins.

Reply:
I just tried this, but it crashes when the editor is started.